plugins/bulk-import/src/components/PreviewFile/PreviewPullRequestForm.tsx

```diff
@@ -49,15 +49,11 @@ export const PreviewPullRequestForm = ({
   setPullRequest,
   formErrors,
   setFormErrors,
-  entityOwner,
-  setEntityOwner,
 }: {
   repoId: string;
   repoUrl: string;
   pullRequest: PullRequestPreviewData;
```

Comment:
Couldn't you use here just one PullRequest Data (type `PulllRequestPreview`) instead of a Record?

Then it should be possible to remove the repoId type?

```suggestion
  pullRequest: PullRequestPreview;
```

Reply:
The form uses a record instead of a single pull request due to the complexity in managing multiple preview forms in a tabbed view for organizations. In this view, it’s necessary to save changes across multiple preview forms and display field errors that might occur in each pull request preview.

Reply:
Got it, thanks